docs/translations/api-docs/table-pagination/table-pagination.json

```diff
@@ -34,15 +35,15 @@
       "nodeName": "the select label Typography element"
     },
     "selectRoot": {
-      "description": "Styles applied to {{nodeName}}.",
+      "description": "@deprecated Styles applied to {{nodeName}}.",
```

Comment:
@oliviertassinari For this CSS API, there is a TODO mention to combine this. Should we mark it as deprecated and remove in the future so we don't have too much breaking change?

Reply:
I think that we should only focus on emotion

Reply:
will add this to breaking change docs